### DISCUSSION TITLE
Can't run rustdesk when using ecryptfs mounted home directory

### DISCUSSION THREAD

**Discussion by frasty** [rustdesk/rustdesk#10296] [General]
### Bug Description

I have my user home directory encrypted with ecryptfs.
After installing rustdesk from rpm package the ecryptfs user's home directory will get unmounted after some random time.

### How to Reproduce

Setup your user home directory as ecryptfs if you don't have already.
Install rustdesk. 
After some random time you will find the user home directory unmounted.
The user home directory will also get unmounted if you manually stop rustdesk service (systemctl stop rustdesk)

### Expected Behavior

rustdesk should not interfere with ecryptfs mounts

### Operating system(s) on local (controlling) side and remote (controlled) side

Fedora Linux 41

### RustDesk Version(s) on local (controlling) side and remote (controlled) side

1.3.5

### Screenshots

![Schermata_20241216_122916](https://github.com/user-attachments/assets/b43b0209-65a2-43f7-b162-4d1f2e5f8220)

**Comment by Abyss777** [comment]
I can confirm the problem.

My encrypted home directory suddenly randomly becomes unmounted and it is related to rustdesk.

Ubuntu 24.04 (KDE Neon)

```
rustdesk --version
1.3.7
```

it is installed from ubuntu deb package

Here is related logs

```
grep -A 7 "home-andrey.mount: Deactivated successfully." /var/log/syslog
2025-02-20T12:19:31.727552+05:00 andrey-nuc systemd[1]: home-andrey.mount: Deactivated successfully.
2025-02-20T12:19:31.744498+05:00 andrey-nuc rustdesk[3762769]: Signature not found in user keyring
2025-02-20T12:19:31.744932+05:00 andrey-nuc rustdesk[3762769]: Perhaps try the interactive 'ecryptfs-mount-private'
2025-02-20T12:19:31.769619+05:00 andrey-nuc kernel: input: RustDesk UInput Keyboard as /devices/virtual/input/input297
2025-02-20T12:19:31.771607+05:00 andrey-nuc kernel: input: mouce-library-fake-mouse as /devices/virtual/input/input298
2025-02-20T12:19:31.862798+05:00 andrey-nuc rustdesk[3762808]: Cannot load libcuda.so.1
2025-02-20T12:19:31.866544+05:00 andrey-nuc rustdesk[3762808]: Cannot load libcuda.so.1
2025-02-20T12:19:31.899159+05:00 andrey-nuc kwin_wayland[3317]: kf.service.sycoca: Couldn't create "/home/andrey/.cache"
```

```
zgrep -A 7 "home-andrey.mount: Deactivated successfully." /var/log/syslog.*.gz
/var/log/syslog.3.gz:2025-01-28T15:22:17.385543+05:00 andrey-nuc systemd[1]: home-andrey.mount: Deactivated successfully.
/var/log/syslog.3.gz-2025-01-28T15:22:17.398205+05:00 andrey-nuc rustdesk[597866]: Signature not found in user keyring
/var/log/syslog.3.gz-2025-01-28T15:22:17.398282+05:00 andrey-nuc rustdesk[597866]: Perhaps try the interactive 'ecryptfs-mount-private'
/var/log/syslog.3.gz-2025-01-28T15:22:17.429843+05:00 andrey-nuc kernel: input: RustDesk UInput Keyboard as /devices/virtual/input/input973
/var/log/syslog.3.gz-2025-01-28T15:22:17.441848+05:00 andrey-nuc kernel: input: mouce-library-fake-mouse as /devices/virtual/input/input974
/var/log/syslog.3.gz-2025-01-28T15:22:17.544589+05:00 andrey-nuc rustdesk[597912]: Cannot load libcuda.so.1
/var/log/syslog.3.gz-2025-01-28T15:22:17.546967+05:00 andrey-nuc rustdesk[597912]: Cannot load libcuda.so.1
/var/log/syslog.3.gz-2025-01-28T15:22:17.579016+05:00 andrey-nuc kwin_wayland[3253]: kf.service.sycoca: Couldn't create "/home/andrey/.cache"
```

**Reply by frasty** [reply]
Indeed, it seems that rustdesk do some very nasty things on systems running ecryptfs. 
tl;dr rustdesk doesn't seem to work with ecryptfs encrypted home dirs. A warning should be put somewhere telling the lack of ecryptfs support.
